pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -139,6 +139,7 @@ func (plugin *kubenetNetworkPlugin) Event(name string, details map[string]interf
 		cidr.IP.To4()[3] += 1
 
 		json := fmt.Sprintf(NET_CONFIG_TEMPLATE, BridgeName, plugin.MTU, network.DefaultInterfaceName, podCIDR, cidr.IP.String())
+		glog.V(2).Infof("CNI network config set to %v", json)
```

Comment:
This happens on every Event, it so happens that the only event today is pod cidr. Upto you we can always dumb it down later. Unsure if a customer is going to care about the details of pod networking, though.